staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -258,6 +261,38 @@ func TestProxyHandler(t *testing.T) {
 			},
 			expectedStatusCode: http.StatusServiceUnavailable,
 		},
+		"change aggregator http request host": {
+			user: &user.DefaultInfo{
+				Name:   "username",
+				Groups: []string{"one", "two"},
+			},
+			path: "/request/path",
```

Comment:
better to explicitly set: serviceResolver: &mockedRouter{destinationHost: xyz}

Reply:
i think destinationHost cannot be set to a fixed value, for example: 127.0.0.1:9999, because the subsequent processing logic will actually call 127.0.0.1:9999. 
So, destinationHost can only be set to: targetServer.Listener.Addr().String()

  
serviceResolver = &mockedRouter{destinationHost: targetServer.Listener.Addr().String()}